src/server/game/Entities/Object/Object.cpp

```diff
@@ -572,8 +572,8 @@ void Object::BuildMovementUpdate(ByteBuffer* data, uint32 flags) const
         {
             std::vector<Position> splines = areaTrigger->GetSplines();
 
-            *data << float(areaTrigger->GetTotalDuration());    // TimeToTarget
-            *data << float(areaTrigger->GetTimeSinceCreated()); // ElapsedTimeForMovement
+            *data << float(areaTriggerMiscTemplate->TimeToTarget);
+            *data << float(0); // ElapsedTimeForMovement, always 0 because sent at creation
```

Comment:
This is a wrong assumption, a player entering visibility range of the areatrigger after it has been created will see a nonzero value